projects/1.16/assets/dungeons-gear/dungeons_gear/lang/zh_cn.json

```
@@ -49,6 +55,7 @@
   "enchantment.dungeons_gear.critical_hit": "暴击",
   "enchantment.dungeons_gear.critical_hit.desc": "有几率造成非常规的暴击伤害。",
   "enchantment.dungeons_gear.echo": "回击",

```

```suggestion
  "enchantment.dungeons_gear.echo": "回音",

```
感觉应该是回音吧，回击完全变成另一个意思了。